src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/services/service-form/service-form.component.html

```diff
@@ -83,15 +83,54 @@
             <span class="invalid-feedback"
                   *ngIf="serviceForm.showError('service_id', frm, 'uniqueName')"
                   i18n>This service id is already in use.</span>
-            <span class="invalid-feedback"
-                  *ngIf="serviceForm.showError('service_id', frm, 'rgwPattern')"
-                  i18n>The value does not match the pattern <strong>&lt;service_id&gt;[.&lt;realm_name&gt;.&lt;zone_name&gt;]</strong>.</span>
             <span class="invalid-feedback"
                   *ngIf="serviceForm.showError('service_id', frm, 'mdsPattern')"
                   i18n>MDS service id must start with a letter and contain alphanumeric characters or '.', '-', and '_'</span>
           </div>
         </div>
 
+        <div class="form-group row"
+             *ngIf="serviceForm.controls.service_type.value === 'rgw'">
+          <label class="cd-col-form-label"
+                 for="realm_name">
+            <span i18n>Realm Name</span>
+          </label>
+          <div class="cd-col-form-input">
+            <input id="realm_name"
+                   class="form-control"
+                   type="text"
+                   formControlName="realm_name">
+          </div>
+        </div>
+
+        <div class="form-group row"
+             *ngIf="serviceForm.controls.service_type.value === 'rgw'">
+          <label class="cd-col-form-label"
+                 for="zonegroup_name">
+            <span i18n>Zonegroup Name</span>
+          </label>
+          <div class="cd-col-form-input">
+            <input id="zonegroup_name"
+                   class="form-control"
+                   type="text"
+                   formControlName="zonegroup_name">
+          </div>
+        </div>
+
+        <div class="form-group row"
+             *ngIf="serviceForm.controls.service_type.value === 'rgw'">
+          <label class="cd-col-form-label"
+                 for="zone_name">
+            <span i18n>Zone Name</span>
+          </label>
+          <div class="cd-col-form-input">
+            <input id="zone_name"
+                   class="form-control"
+                   type="text"
+                   formControlName="zone_name">
+          </div>
```

Comment:
Lets set the default value to "default" for zone and zonegroup, that's what is used.

Reply:
Also add some helper text to each one of them

Reply:
Done